packages/checkout/blocks-registry/index.ts

```diff
@@ -175,18 +175,28 @@ export const registerCheckoutBlock = (
 	assertOption( options, 'areas', 'array' );
 	assertBlockComponent( options, 'component' );
 
+	/**
+	 * If provided with a configuration object, this registers the block with WordPress.
+	 */
 	if ( options?.configuration ) {
 		assertOption( options, 'configuration', 'object' );
 		registerExperimentalBlockType( blockName, {
 			...options.configuration,
 			category: 'woocommerce',
+			parent: [],
```

Comment:
Can't we extrapolate this from `options.areas`?

Reply:
It doesn't seem like we need to. Those areas still work.

Reply:
I'm not sure I understand?
I mean that instead of waiting for 3PD to pass the parent value, we implicitly provide it from what's in `options.areas`

Reply:
Parent _should_ be empty. We're not accepting this prop from 3PD, it's being overwritten as `[]`.

3PD use the area prop to define where they want blocks to appear. Not parent.

Reply:
Ah right, sorry. My understanding is that having `[]` would disable the block from all inserters.
I don't understand yet how can we show sample block only on our innerBlocks, is it because `allowedBlocks` override `parent`?

Reply:
Yup it certainly looks that way. My testing instructions cover this—sample block can be inserted into the contact information section.

Reply:
Yeah, I can see that. Approved.